udata/templates/discussion/threads.html

```diff
@@ -9,55 +9,57 @@
 #}
 {% set threadclass = 'col-sm-9' if sidetext else 'col-sm-12' %}
 <div class="row">
-    <div class="{{ threadclass }} list-group resources-list smaller">
-    {% for discussion in discussions %}
-        {% include theme('discussion/item.html') %}
-    {% endfor %}
-        <a class="list-group-item add new-discussion"
-            href="#">
-            <div class="format-label pull-left">+</div>
-            <h4 class="list-group-item-heading">
-                {{ _('Start a new discussion') }}
-            </h4>
-        </a>
-        {% set discussion_length = discussions|rejectattr('closed')|list|length %}
-        <div class="list-group-item list-group-form list-group-form-discussion hidden animated" id="discussion-{{ discussion_length + 1 }}">
-            <div class="format-label pull-left">
-                {{ current_user|avatar(52) }}
-            </div>
-            <span class="list-group-item-link">
-                <a href="#discussion-{{ discussion_length + 1 }}"><span class="fa fa-link"></span></a>
-            </span>
-            <h4 class="list-group-item-heading">
-                {{ _('Starting a new discussion thread') }}
-            </h4>
-            <p class="list-group-item-text">
-                {{ _("You're about to start a new discussion thread. Make sure that a thread about the same topic doesn't exist yet just above.") }}
-                {% if subtext %}{{subtext}}{% endif %}
-            </p>
-            <form role="form" class="clearfix animated"
-                data-api-url="{{ url_for('api.discussions', for=subject.id) }}"
-                data-subject-class="{{ subject.__class__.__name__ }}"
-                data-subject-id="{{ subject.id }}">
-                <div class="form-group">
-                    <label for="title-new-discussion">{{ _('Title') }}</label>
-                    <input type="text" id="title-new-discussion" name="title" class="form-control" required />
-                    <label for="comment-new-discussion">{{ _('Comment') }}</label>
-                    <textarea id="comment-new-discussion" name="comment" class="form-control" rows="3" required></textarea>
+    <div class="{{ threadclass }} resources-list smaller">
```

Comment:
Is this supposed to be in this PR? Seems unrelated to cards.

Reply:
conflict resolution